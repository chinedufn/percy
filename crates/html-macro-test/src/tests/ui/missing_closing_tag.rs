//! # To Run
//!
//! cargo test -p html-macro-test -- ui trybuild=missing_closing_tag.rs

extern crate percy_dom;
use percy_dom::prelude::*;

// Open tag that are missing their corresponding closing tags.
fn main() {
    // One missing
    html! {
        <div>
    };

    // One missing with one valid
    html! {
        <strong>
          <span></span>
    };

    // Two missing
    html! {
        <em>
          <footer>
    };
}
