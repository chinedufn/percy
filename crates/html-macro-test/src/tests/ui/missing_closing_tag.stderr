error: Missing closing tag.
  --> src/tests/ui/missing_closing_tag.rs
   |
   |         <div>
   |          ^^^

error: Missing closing tag.
  --> src/tests/ui/missing_closing_tag.rs
   |
   |         <strong>
   |          ^^^^^^

error: Missing closing tag.
  --> src/tests/ui/missing_closing_tag.rs
   |
   |         <em>
   |          ^^

error: Missing closing tag.
  --> src/tests/ui/missing_closing_tag.rs
   |
   |           <footer>
   |            ^^^^^^
